test/core/primitives/primitives_codec_test.cpp

```diff
@@ -134,21 +133,40 @@ class Primitives : public testing::Test {
                            {{4, 5}, {6, 7, 8}},  // provides
                            2};                   // longivity
   ByteArray encoded_valid_transaction_{
-      1,                       // variant type order
-      1, 0, 0, 0, 0, 0, 0, 0,  // priority
-      8,                       // compact-encoded collection size (2)
+      1,  // variant type order
```

Comment:
I think it's better to stick to the original formatting.

Reply:
clang-format did it. think it better to add //clang-format off/on ?